frontend/public/components/RBAC/_rbac.scss

```diff
@@ -1,7 +1,3 @@
-.rbac-rules-list {
```

Comment:
Remove the [className from the jsx](https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/rules.jsx#L14), too?